roles/tests-integration/tests/common/mod.rs

```diff
@@ -396,9 +396,9 @@ pub async fn start_sv2_translator(upstream: SocketAddr) -> SocketAddr {
     .expect("failed");
     let listening_address = get_available_address();
     let listening_port = listening_address.port();
-    let hashrate = measure_hashrate(3) as f32 / 100.0;
+    let hashrate = measure_hashrate(3) as f32 / 20.0;
```

Comment:
I did test many times, and some of them it was failing. So I tried to change this in:
```suggestion
    let hashrate = measure_hashrate(1) as f32 / 20.0;
```
With that, it never failed again. Don't know if it's random or not btw :)

Reply:
tbh I still don't understand why we are dividing the measured hashrate

that will make tProxy set a very low difficulty and make CPU miner submit way more shares than necessary